dashboard/lib/policies/child_account.rb

```diff
@@ -0,0 +1,57 @@
+class Policies::ChildAccount
+  # Values for the `child_account_compliance_state` attribute
+  module ComplianceState
```

Comment:
Minor nitpick: would this be clearer as ComplianceStatus? Since we also deal with geographic states, it might be helpful to differentiate the two concepts.

Reply:
Oh I really like that name, but I would like this to match the name of the serialized attribute which is `child_account_compliance_state`. And this attribute is already live, so renaming it would be very challenging.